docker/base/jfrog

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+set -xe
+
+VERSION_JFROG=${1:-$VERSION_JFROG}
+
+curl -XGET -L -k "https://api.bintray.com/content/jfrog/jfrog-cli-go/${VERSION_JFROG}/jfrog-cli-linux-amd64/jfrog?bt_package=jfrog-cli-linux-amd64" > /usr/local/bin/jfrog
```

Comment:
`jq -r '.[0].sha256'` could be used to extract the info more reliably (if we include the `jq` tool, which might be useful to have).